packages/module/src/Feedback/FeedbackModalInternal.tsx

```diff
@@ -94,12 +89,12 @@ export const FeedbackModalInternal = memo(({ email, isOpen, onShareFeedback, onJ
               </Card>
               <br />
               {onJoinMailingList ?
-              <Card isSelectableRaised isCompact onClick={() => {typeof onJoinMailingList ==='string' ? window.open(onJoinMailingList, '_blank') : setModalPage('informDirection')}}>
-                <CardTitle className="chr-c-feedback-card-title">
-                  <Text>{intl.informDirection} {typeof onJoinMailingList === 'string' ? <ExternalLinkAltIcon />: null}</Text>
-                </CardTitle>
-                <CardBody>{intl.learnAboutResearchOpportunities}</CardBody>
-              </Card>: null }
+                <Card isSelectableRaised isCompact onClick={() => { typeof onJoinMailingList === 'string' ? window.open(onJoinMailingList, '_blank') : setModalPage('informDirection') }}>
+                  <CardTitle className="chr-c-feedback-card-title">
+                    <Text>{intl.informRedhatDirection} {typeof onJoinMailingList === 'string' ? <ExternalLinkAltIcon /> : null}</Text>
```

Comment:
Looks like the build error is coming from here?

Reply:
@wise-king-sullyman Fixed it thanks.